packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js

```diff
@@ -66,7 +66,25 @@ export const onRenderBody = (
   const setComponents = pluginOptions.head
     ? setHeadComponents
     : setPostBodyComponents
-  return setComponents([
+
+  const inlineScripts = []
+  if (pluginOptions.enableWebVitalsTracking) {
+    // web-vitals/polyfill (necessary for non chromium browsers)
+    // @seehttps://www.npmjs.com/package/web-vitals#how-the-polyfill-works
+    inlineScripts.push(
```

Comment:
by default this will put polyfill at the end of `<body>` - can this cause a problems / should we make sure it is added to `<head>`? (tagmanager seems to always add stuff to `<head>`, so this only applies to `ga`)

Ref: 
https://github.com/gatsbyjs/gatsby/blob/d06d6b5a286bc6140f178d30170674e755b5f273/packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js#L66-L68

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-google-analytics/src/gatsby-node.js#L9-L13